packages/eslint-plugin-pf-codemods/lib/rules/v5/cardHeader-update-api.js

```diff
@@ -0,0 +1,139 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8759
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
+    const cardImports = getPackageImports(
+      context,
+      "@patternfly/react-core"
+    ).filter((specifier) =>
+      ["CardHeaderMain", "CardActions", "CardHeader"].includes(
+        specifier.imported.name
+      )
+    );
+
+    return !cardImports.length
+      ? {}
+      : {
+          ImportDeclaration(node) {
+            if (
+              node.source.value !== "@patternfly/react-core" ||
+              !node.specifiers.filter((specifier) =>
+                cardImports
+                  .map((imp) => imp.imported.name)
+                  .includes(specifier?.imported?.name)
+              ).length
+            ) {
+              return;
+            }
+
+            node.specifiers
+              .filter((specifier) =>
+                ["CardHeaderMain", "CardActions"].includes(
+                  specifier.imported.name
+                )
+              )
+              .forEach((specifier) => {
+                const hasCommaAfter =
+                  context.getSourceCode().getTokenAfter(specifier).value ===
+                  ",";
+
+                context.report({
+                  node,
+                  message: `${specifier.imported.name} is no longer exported.`,
+                  fix(fixer) {
+                    return hasCommaAfter
+                      ? fixer.removeRange([
+                          specifier.range[0],
```

Comment:
Could you make it so that the trailing comma is also removed (if there is one I guess?)

I wouldn't oppose this being a followup if you'd like to do it under a new issue.